pylint_odoo/checkers/modules_odoo.py

```diff
@@ -484,6 +485,16 @@ def _get_duplicate_xml_fields(self, fields):
             all_fields.setdefault(field_xml, []).append(field)
         # Remove all keys which not duplicated
         for key, items in all_fields.items():
+            parents = [e.getparent() for e in items]
+            same_parent = [
+                item for item, count in collections.Counter(parents).items()
+                if count > 1]
```

Comment:
> Are you using a beta version or something?

Yes I'm, I added this mccabe behaviour for pylint 2.0.0 (unreleased yet) but you can install it with:
`pip install --upgrade git+https://github.com/PyCQA/astroid.git@master#egg=astroid && git+https://github.com/PyCQA/pylint@master#egg=pylint`